SConstruct

```diff
@@ -1360,6 +1360,8 @@ libraries = {
 			"LIBPATH" : [ "$RENDERMAN_ROOT/lib" ],
 		},
 		"pythonEnvAppends" : {
+			"CPPDEFINES" : [ "RMAN_RIX_NO_WARN_DEPRECATED" ],
+			"CPPPATH" : [ "$RENDERMAN_ROOT/include" ],
```

Comment:
To build on Linux I needed to update "LIBS" below to include `"IECoreScene$CORTEX_LIB_SUFFIX"`